components/landing/QuickStart/StartLocally.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+  <section class="start-locally">
+    <h3 class="start-locally__title">
+      Start locally
+    </h3>
+    <p class="start-locally__description">
+      Although it isn’t required we recommend using
+      <AppLink
+        class="start-locally__link"
+        :url="downloadAnaconda.url"
+      >
+        {{ downloadAnaconda.label }}
+      </AppLink>
+      to cleanly separate Qiskit from other applications and improve your experience.
+    </p>
+    <div class="start-locally__options">
+      <div
+        v-for="optionsBlock in installOptions"
+        :key="optionsBlock.alias"
+      >
+        <h4 class="start-locally__option-title">
+          {{ optionsBlock.title }}
+        </h4>
+        <cv-button-set :class="`start-locally__options-group start-locally__options-group_${optionsBlock.alias}`">
+          <cv-button
+            v-for="option in optionsBlock.elements"
+            :key="option"
+            :title="option"
+            class="start-locally__option"
+            :class="{ 'start-locally__option_active': isActive (optionsBlock.alias, option) }"
+            @click="selectOption(optionsBlock.alias, option)"
```

Comment:
use it here too:
```suggestion
            @click="selectOption(optionsBlock.alias, index)"
```
to set the selected index.